module/account/src/Infrastructure/Handler/UpdateUserCommandHandler.php

```diff
@@ -39,13 +51,19 @@ public function __invoke(UpdateUserCommand $command)
     {
         $user = $this->repository->load($command->getId());
         Assert::notNull($user);
+
         $user->changeFirstName($command->getFirstName());
         $user->changeLastName($command->getLastName());
         $user->changeLanguage($command->getLanguage());
         $user->changeRole($command->getRoleId());
+
         if ($command->getPassword()) {
-            $user->changePassword($command->getPassword());
+            $encodedPassword = $this->userPasswordEncoder->encodePassword($user, $command->getPassword()->getValue());
+            $password = new Password($encodedPassword);
```

Comment:
Add 
```
$encodedPassword = $this->userPasswordEncoder->encodePassword($user, $command->getPassword()->getValue());
$password = new Password($encodedPassword);
```
to new Service class along with the interface, name it somethink like Encoding/hashing service and us it in `CreateUserCommandHandler` to